index.rst

```diff
@@ -48,14 +48,480 @@
+Introduction
+============
 
-.. .. rubric:: References
+Some of the earliest applications of image differencing in astronomy were motivated by the desire to identify transient sources (microlensing events) in extremely crowded stellar fields.
+At ground-based photometric precision, typically about one percent of stellar sources have detectable variability. 
+Thus by subtracting a coadded historical (template) image from a new (science) image, the crowding can be reduced by a factor of ~100.
+
+In this technote we assess the state of crowded field processing in Fall 2020 using the LSST Alert Production (AP) Science Pipelines.
+Alert Production uses image differencing to identify transient, moving, and variable sources in LSST images within 60 seconds of camera readout.
+Crucially, the AP processing must meet its requirements even in the most crowded fields it will encounter.
+Of particular importance are the transient reporting latency (OTT1 and OTR1; LSR-REQ-0117 and LSR-REQ-0025), 
+the reliability with which images are processed (sciVisitAlertDelay and sciVisitAlertFailure; OSS-REQ-0112), 
+the photometric and astrometric accuracy of the resulting catalogs (dmL1AstroErr, dmL1PhotoErr, and photoZeroPointOffset; OSS-REQ-0149 and OSS-REQ-0152), 
+completeness and purity metrics for transients (OSS-REQ-0353) and solar system objects ( OSS-REQ-0354),
+and source misassociation for transients (OSS-REQ-0160) and solar system objects (OSS-REQ-0159).
+Accordingly, large-scale tests on precursor datasets are necessary to identify where improvements are needed in order to meet requirements.
+The status of the processing also will inform ongoing cadence planning for the overall survey.
+
+The scope of this technote does not include obtaining photometry for static sources in direct images (except insofar as it is required to conduct image differencing), nor comparisons to dedicated crowded field processing codes.
+`DMTN-077 <https://dmtn-077.lsst.io>`_ addressess these issues.
+
+Overview of the data
+====================
+
+To test the LSST Science Pipelines on crowded fields, we acquired public data from the DECam survey of the galactic bulge by `Saha et. al <https://arxiv.org/pdf/1902.05637.pdf>`_.
+The full survey consists of six fields at low galactic latitudes observed in the six ``ugrizy`` bands, though for this investigation we limited the analysis to g- and i-bands for the two fields closest to the galactic center (B1 and B2, :numref:`field_coordinates`).
+The two fields selected for this test were observed in 2013 and 2015 to similar depth in both bands (:numref:`visit_summary`).
+These data are available on ``lsst-devl`` at ``/datasets/decam``, while the processed data and the scripts that produced them available at ``/project/sullivan/saha2``.
+The size of the processed data amounts to 7.0Tb on disk, which can be fully recreated following the instructions in ``/project/sullivan/saha2/README``.
```

Comment:
I've added a reference to the software version I used (release 21.0), but I am unsure how to best include information from the README. I can see how someone reading this technote might be curious and want to read it, but not have access to lsst-devl, but the README itself doesn't make much sense out of context since it refers to directories and scripts in that location. For now, I've tweaked the wording to make it more clear that the README gives instructions on how to run the scripts.

Reply:
I would suggest that all the `.conf`, `.sh`, and `.sl` files are worthy of being preserved in GitHub, either in the DMTN repo or a separate `lsst-dm` repo.  The few references to `lsst-devl`-specific locations (primarily `/software/lsstsw`, `/datasets/decam`, and `/project/sullivan/saha2` itself, I believe) can be marked or an environment variable substituted.  References to `/project/sullivan/code/ip_isr` (if actually needed) should instead refer to a particular release or at least GitHub hash.